src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -535,6 +535,7 @@ def load_arguments(self, _):
         c.argument('network_dataplane', arg_type=get_enum_type(network_dataplanes))
         c.argument('network_policy')
         c.argument('network_plugin', arg_type=get_enum_type(network_plugins))
+        c.argument('ip_families')
```

Comment:
If the value of this param is a list, please use `nargs='+'` to define it.
```suggestion
        c.argument('ip_families', nargs='+')
```

Reply:
@yanzhudd - would that be correct here? We treat the param as a string and then parse it here https://github.com/Azure/azure-cli-extensions/blob/850c41eb2fdea09fd30828a4047362a866c6bc93/src/aks-preview/azext_aks_preview/managed_cluster_decorator.py#L280C100-L280C100

Reply:
Hi @tyler-lloyd,
yeah, your logic works. But it is recommented to use `nargs='+'` to define parameters whose value is a list. After defining with that, the input will be automatically separated by spaces and you don't need to parse it manually.

Reply:
@yanzhudd - please review the last commit. is this what you're talking about? https://github.com/Azure/azure-cli-extensions/pull/7040/commits/d7ea54a3595a735b2ca8677ceb2b2609f7bc44f1